Cross update

Evankhell561 opened the pull request:
#What Does This PR Do
añade las galletas blue, yellow, metal, adamantina y silver del slime cross asi como otros pocos cross extra y refactoriza las ganancias energeticas dadas por otros efectos del cruce de slimes.

#Why It's Good For The Game
Por un slime cross cada vez más completo

Images of changes
## Changelog
:cl:
add: crossgalleta azul
add: crossgalleta metalica
add: crossgalleta amarilla
add: crossgalleta adamantina
add: cross estabilizado azul
add: trait noslipwater
add: cross estabilizado plateado
add: reactivo nutriente estabilizado
twaak: cross energy refactor
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Evankhell561 committed:
cross recuperados

Evankhell561 committed:
balance

Evankhell561 committed:
Merge branch 'Lambda' into cross_update

Evankhell561 committed:
fix

Evankhell561 committed:
Merge branch 'Lambda' into cross_update

Silverexperience reviewed (changes requested):
- Las galletas que te hacen inmune a resbalarte e invocan agua a tu arededor, al final cuando se desactivan puedes resbalarte porque el efecto de inmunidad termina antes que el de invocar agua.

- El core azul stabilizado tarda un rato en aplicarse y puedes resbalarte en esos instantes.

- Los efectos de tener mas resistencia al fuego parece que no se pierden.
![imagen](https://user-images.githubusercontent.com/58746682/97810478-6880fd00-1c74-11eb-94b5-9f1fe6495785.png)

- ¿Como funciona lo de cross energy refactor?

Evankhell561 commented:
> * ¿Como funciona lo de cross energy refactor?

basicamente ahora la ganancia de energia de esos slimes es una ganacia plana y no porcentual. queda igual para las baterias basicas pero las mejoradas recibiran menos energia en proporcion.

Evankhell561 committed:
fixes1, aun no esta listo

Evankhell561 commented:
> * La  el core azul stabilizado tarda un rato en aplicarse y puedes resbalarte en esos instantes.

esto es... muy raro porque se aplica al instante.

Evankhell561 committed:
rever vars ineccesarias

puse esto porque pense que funcionaba como en eris pero segun veo en el code no es asi asi que lo retiro.

Evankhell561 committed:
Merge branch 'Lambda' into cross_update

Evankhell561 committed:
Merge branch 'Lambda' into cross_update

Evankhell561 commented:
confictos

Evankhell561 committed:
Merge branch 'Lambda' into cross_update

Evankhell561 committed:
Update _status_effect.dm

Evankhell561 committed:
adamantinecookie

Evankhell561 committed:
fixes

Evankhell561 committed:
Update consuming.dm

Evankhell561 committed:
Update consuming.dm

Evankhell561 commented:
> Las galletas que te hacen inmune a resbalarte e invocan agua a tu arededor, al final cuando se desactivan puedes resbalarte porque el efecto de inmunidad termina antes que el de invocar agua.

fixed

Evankhell561 commented:
Ya arreglé todos los problemas menos el  que causa que puedas resbalarte en 0.1s segundos antes de aplicar el efecto como tal y no puedo solucionar eso, sin embargo, ya arreglé todo lo demás y mucho más.

Evankhell561 commented:
Tiempo limite cumplido, avisé por discord.

Evankhell561 commented:
Agradecer también a @Silverexperience por darse cuenta de errores que de otra forma no habria podido notar. se te quiere mucho.
